language/af.json

```diff
@@ -0,0 +1,99 @@
+{
+  "semantics": [
+    {
+      "label": "Opskrif",
+      "placeholder": "Hier kan jy 'n opsionele opskrif byvoeg.",
+      "description": "Die opskrif wat jy graag bo die prent wil vertoon"
+    },
+    {
+      "label": "Items",
+      "entity": "item",
+      "widgets": [
+        {
+          "label": "Verstek"
+        }
+      ],
+      "field": {
+        "label": "Item",
+        "fields": [
+          {
+            "label": "Beeld"
+          },
+          {
+            "label": "Etiket",
+            "description": "Optionele etiket vir 'n regmerkie. Maak asb. seker dat dit nie te lank is nie, anders sal dit versteek word."
+          },
+          {
+            "label": "Beskrywing",
+            "placeholder": "My prent beskrywing ...",
+            "description": "Optionele beskrywing vir die prent"
+          },
+          {
+            "label": "Klank",
+            "description": "Opsionele klankbaan wat speel wanneer 'n prent vertoon word."
+          }
+        ]
+      }
+    },
+    {
+      "label": "Gedragsinstellings",
+      "description": "Hierdie opsies laat jou beheer uitoefen oor die uitvoering van die taak.",
+      "fields": [
```

Comment:
@weblate Those missing labels would probably leave the users a little lost.

Reply:
Yes, there was a bug when updating Weblate that set some string to read only. I've fixed it and I'm giving Victor some time to finish the translations in Weblate.

Reply:
Perfect! Then I'll just wait for the next push to this repository.